spec/spec_helper.rb

```diff
@@ -35,6 +35,7 @@
   config.include GdsApi::TestHelpers::AssetManager
   config.include GovukSchemas::RSpecMatchers
   config.include AuthenticationHelper, type: :feature
+  config.include AuthenticationHelper, type: :request
```

Comment:
You'll have to change lines 56 - 58 too: https://github.com/alphagov/content-publisher/pull/1504/commits/b5d4f17064dfbfbd12e8b6cfa1dc7176e05f0210#diff-93830fa29d616f7c87903d08b5b1b29aR56-R58 

Here's a snippet from my branch as I encountered the same thing when creating some requests tests (yours will probs be merged before mine) 😄 -
```
  config.after :each, type: lambda { |s| [:feature, :request].include?(s) } do
    reset_authentication
  end
```